packages/cli/src/services/project-parser.ts

```diff
@@ -193,7 +190,7 @@ async function findFilesWithPattern (
   ignorePattern: string[],
 ): Promise<string[]> {
   // The files are sorted to make sure that the processing order is deterministic.
-  const files = await globPromise(pattern, {
+  const files = await glob(pattern, {
```

Comment:
Is this a promise now?

Reply:
That's what the docs are saying 
```
// load using import
import { glob, globSync, globStream, globStreamSync, Glob } from 'glob'
// or using commonjs, that's fine, too
const {
  glob,
  globSync,
  globStream,
  globStreamSync,
  Glob,
} = require('glob')

// the main glob() and globSync() resolve/return array of filenames

// all js files, but don't look in node_modules
const jsfiles = await glob('**/*.js', { ignore: 'node_modules/**' })
```
https://www.npmjs.com/package/glob